bokehjs/src/compiler/prelude.ts

```diff
@@ -219,7 +219,7 @@ ${comment(license)}
 })(this, function(Bokeh, version) {
   let define;
   return (function(modules, entry, aliases, externals) {
-    const bokeh = typeof Bokeh !== "undefined" && (version != null ? Bokeh[version] : Bokeh);
+    const bokeh = (typeof Bokeh !== "undefined" && version != null ? Bokeh[version] : Bokeh;
```

Comment:
maybe I am missing something obvious, I don't see where this open paren is closed

Reply:
Just a mistake, though that logic wasn't good anyway.